Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -37,7 +37,7 @@ public class SiteExtensionManager : ISiteExtensionManager
         private readonly IDeploymentSettingsManager _settings;
         private readonly ITraceFactory _traceFactory;
 
-        private const string _applicationHostFile = "applicationHost.xdt";
+        private const string _applicationHostFile = Constants.ApplicationHostXdtFileName;
```

Comment:
Why set a constant to another constant, instead of using `Constants.ApplicationHostXdtFileName` directly?